Dockerfile

```
@@ -11,6 +11,10 @@ USER ruby
 
 COPY --chown=ruby:ruby Gemfile* ./
 RUN gem install bundler -v 2.4.10
+
+RUN bundle config set --local without development:test \
+  && bundle config set --local jobs "$(nproc)"
+
 RUN bundle install --jobs "$(nproc)"
```

Specifying jobs isn't necessary if it's been configured

```suggestion
RUN bundle install"
```

---

Thanks Laurence, missed that this time round 👍